attributes/default.rb

```diff
@@ -1,5 +1,6 @@
 # This cookbook support only runit
 default['bosun']['init_style']      = 'runit'
+default['bosun']['install_style']   = 'binary' # binary or golang
```

Comment:
Maybe it would be better to use attribute version (latest = golang, else binary) for that?